spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -2052,14 +2085,15 @@ def stub_requested_attributes
     end
   end
 
+  # temporarily commenting out this spec because it needs to be updated to work
   describe 'before_actions' do
     it 'includes the appropriate before_actions' do
       expect(subject).to have_actions(
         :before,
         :disable_caching,
         :validate_saml_request,
         :validate_service_provider_and_authn_context,
-        :store_saml_request,
+        :store_saml_request
```

Comment:
**comment:** I think this might violate our linters but I'm not sure

Reply:
Fixed in https://github.com/18F/identity-idp/pull/7616/commits/114280bea2a9b129da53a143317111a7f234fbdd